pkg/common/helpers/parser.go

```diff
@@ -0,0 +1,15 @@
+package helpers
+
+import "strings"
```

Comment:
you might want to change the file name to aws.go or awsparser.go

Reply:
Verified that it has been renamed to aws.go